students/10th/yeajinpark/login.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+      <div class = "background" >
+        <img src = "img/logo_text.png">
```

Comment:
```suggestion
  <body>
    <div class = "background" >
      <img src = "img/logo_text.png">
```

부모 자식 태그 간의 관계는 띄어쓰기 두 칸으로 표기해주세요.

참고로 vscode를 사용하신다면 Ctrl + K, F(windows), Shift + Option + F(Mac OS)로 자동 줄맞춤을 해줄 수 있습니다.

Reply:
꿀팁 감쟈합니다.